docs/source/datamanagement/discovery_storage.md

```diff
@@ -30,25 +32,22 @@ The `/scratch` space is only for temporary storage; this storage is not backed u
 - **Default Permission:** By default, users are given read and write access when added to `/work`. However, specific permissions might be granted at the PI’s request.
 
 
-:::{attention}
-The `/research` storage tier is no longer provided. Please contact Research Computing if you are a former user of `/research` and have questions or issues related to `/research` by [submitting a ticket]. Other storage options include `/work`, [Sharepoint], and [OneDrive].
-:::
-
 ##  Archival Storage
 
 :::{important}
-If you are not connected to the campus internet, you must be connected to the university's {term}`VPN` (i.e., GlobalProtect) before accessing the `/nese` system. You can find detailed information about downloading and using the GlobalProtect VPN in the [FAQ: VPN and remote access].
+If you are not connected to the campus internet, you must be connected to the university's {term}`VPN` (i.e., GlobalProtect) before accessing the `/nese` system. You can find detailed information about downloading and using the GlobalProtect VPN in the [How do I connect to Northeastern VPN?].
```

Comment:
This section can be deleted. nese is now accessed via globus and there is no need for a VPN (also the VPN link is broken, but we don't need it)